src/Microsoft.VisualStudio.ProjectSystem.VisualBasic.VS/ProjectSystem/VS/Automation/VSImports.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.ComponentModel.Composition;
+using System.Linq;
+using EnvDTE;
+using Microsoft.VisualStudio.ProjectSystem.VS.ConnectionPoint;
+using VSLangProj;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Automation
+{
+    [Export(typeof(Imports))]
+    [Export(typeof(ImportsEvents))]
+    [AppliesTo(ProjectCapability.VisualBasic)]
```

Comment:
This doesn't work how you expect it to work. Capabilities do not apply to services (ie single imports). This will also be imported in C#, you'll need a different approach here.